src/chart/rendering/colorbuilder.h

```diff
@@ -0,0 +1,48 @@
+#ifndef COLORBUILDER_H
+#define COLORBUILDER_H
+
+#include "base/anim/interpolated.h"
+#include "base/gfx/color.h"
+#include "base/gfx/colorgradient.h"
+#include "base/gfx/colorpalette.h"
+#include "base/math/range.h"
+#include "chart/generator/colorbase.h"
+
+namespace Vizzu::Draw
+{
+
+struct ColorBuilder
+{
+	using LighnessRange = Math::Range<double>;
+
+	LighnessRange lighnessRange{0.4, -0.4};
+	const Gfx::ColorGradient *gradient{nullptr};
+	const ::Anim::Interpolated<Gfx::ColorPalette> *palette{nullptr};
+
+	ColorBuilder() = default;
```

Comment:
Is this default constructor used anywhere? If not, please remove it, and transform members to references (or ref_wrapper)